project/script_TC.tcl

```diff
@@ -3,29 +3,55 @@
 #   vivado_hls -p trackletCalculator
 # WARNING: this will wipe out the original project by the same name
 
+set modules_to_test {
+  {TC_L1L2E}
+  {TC_L1L2F}
+  {TC_L1L2G}
+  {TC_L1L2H}
+  {TC_L1L2I}
+  {TC_L3L4C}
+  {TC_L3L4D}
+  {TC_L3L4E}
+  {TC_L3L4F}
+}
+# module_to_export must correspond to the default macros set at the top of the
+# test bench; otherwise, the C/RTL cosimulation will fail
+set module_to_export TC_L1L2E
+
 # create new project (deleting any existing one of same name)
 open_project -reset trackletCalculator
 
 # source files
 set CFLAGS {-std=c++11 -I../TrackletAlgorithm}
-set_top TrackletCalculator_L1L2G
 add_files ../TrackletAlgorithm/TrackletCalculatorTop.cc -cflags "$CFLAGS"
-add_files -tb ../TestBenches/TrackletCalculator_L1L2G_test.cpp -cflags "$CFLAGS"
-
-open_solution "solution1"
-
-# Define FPGA, clock frequency & common HLS settings.
-source settings_hls.tcl
```

Comment:
it seems like this test bench no longer sources the common `settings_hls.tcl` -- is this obsolete?

Reply:
It does, down on line 44:
https://github.com/cms-L1TK/firmware-hls/blob/9cede70f59afb6a19e6c8f021fb42794e802f14d/project/script_TC.tcl#L44

It needs to do this inside the loop because the clock has to be created for each solution.